File: 0153-find-minimum-in-rotated-sorted-array/0153-find-minimum-in-rotated-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMin = function(nums) {
    let min = nums[0];
    for(let i=0;i<nums.length;i++){
        if(min>nums[i]){
            min = nums[i]
        }
    }
    return min
};

File: 0153-find-minimum-in-rotated-sorted-array/0153-find-minimum-in-rotated-sorted-array.py
class Solution(object):
    def findMin(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        min_ = nums[0]
        for i in range(0,len(nums)):
            if min_>nums[i]:
                min_ = nums[i]
        return min_